admin-dev/themes/new-theme/js/pages/module/controller.js

```diff
@@ -1124,6 +1131,43 @@ class AdminModuleController {
       self.updateModuleVisibility();
     });
   }
+
+  updateTotalResults() {
+    const updateText = (element, value) => {
```

Comment:
Can we find something more accurate that `updateText()` ? It looks like something close to `replaceFirstWordBy()` ?

Reply:
Thanks I will change it :D